ibtk/src/utilities/libmesh_utilities.cpp

```diff
@@ -300,8 +301,8 @@ get_local_active_element_bounding_boxes(const libMesh::MeshBase& mesh,
 std::vector<libMeshWrappers::BoundingBox>
 get_local_active_element_bounding_boxes(const libMesh::MeshBase& mesh, const libMesh::System& X_system)
 {
-    const unsigned int dim = mesh.mesh_dimension();
-    TBOX_ASSERT(dim <= LIBMESH_DIM);
+    const unsigned int spacedim = mesh.spatial_dimension();
```

Comment:
`mesh_dim` ?

Reply:
I'm not sure I understand the question: what do you have in mind?

Reply:
The way that I think of it, at least, is that `NDIM` is the "spatial dimension". So I am suggesting:
```suggestion
    const unsigned int mesh_dim = mesh.spatial_dimension();
```

Reply:
I agree with the first part but not the second. `mesh.mesh_dimension()` is the topological dimension of the mesh (which is not what we want): `mesh_dim` looks too much like that so I would prefer to write `spacedim` or `space_dimension`